gradle/libs.versions.toml

```diff
@@ -1,9 +1,11 @@
 # Centralized versions for dependencies that share versions
 [versions]
 androidx_test = "1.4.0-beta01"
+compose = "1.0.0-SNAPSHOT"
+composeSnapshot = "7395546"
```

Comment:
Add a comment explaining where this is sourced from and what the relation to the compose version above is.

Reply:
Sure